scala/core/src/partprops/SchemaElements.scala

```diff
@@ -0,0 +1,115 @@
+package semagrams
```

Comment:
Right now, `Part` is mentioned throughout the library, and depends on `Ob`. I'd be OK making a clearer separation between parts and entities (by which I mean things represented on the screen), but it would take some more work.